lms/djangoapps/teams/static/teams/js/views/team_profile.js

```diff
@@ -78,9 +78,9 @@
                     event.preventDefault();
                     var view = this;
                     ViewUtils.confirmThenRunOperation(
-                        gettext("Leave this team?"),
-                        gettext("If you leave, you can no longer post in this team's discussions. Your place will be available to another learner."),
-                        gettext("Confirm"),
+                        _.escape(gettext("Leave this team?")),
```

Comment:
It seems like every call to confirmThenRunOperation would need to have escape() added to its arguments then.  I think better would be to do the escaping inside confirmThenRunOperation (or PromptView.Warning), so that all arguments will be treated as text rather than HTML.  @andy-armstrong what do you say?